packages/nextjs/app/builders/0xfb8b972b0D1cF9BB242A40F7AEA8E9a1bF89E0E3/page.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Image from "next/image";
+
+const zeze = () => {
+  return (
+    <>
+      <div className="flex flex-row min-h-full">
```

Comment:
You might consider to have a more responsive design using Tailwind CSS to adjust the layout on different screen sizes